modules/acid/src/main/ts/ephox/acid/gui/components/HueSlider.ts

```diff
@@ -29,7 +30,11 @@ const sliderFactory = (translate: (key: string) => string, getClass: (key: strin
       tag: 'div',
       classes: [ getClass('hue-slider') ],
       attributes: {
-        role: 'presentation'
+        'role': 'presentation',
+        'aria-role': 'slider',
```

Comment:
This should just be `role` not sure there is a `aira-role`?

- [ ] Remove `aira-role` replace `role` with `slider`